### PR TITLE
[wayland] Use CREW_OPTIONS

### PR DESCRIPTION
**Pull request by supechicken**
Tested on `x86_64`

### CODE REVIEW

File: packages/wayland.rb [hunk 1]
```diff
@@ -3,38 +3,29 @@
...
+    system "./configure \
+            #{CREW_OPTIONS} \
+            --disable-documentation"
+    system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system 'libtool --finish /usr/local/lib64'
```
Comment (uberhacker) [inline packages/wayland.rb:29]: Is this needed?  If it was, it should be abstracted:
```
system "libtool --finish #{CREW_LIB_PREFIX}"
```

Comment (supechicken) [reply packages/wayland.rb]: Done

File: packages/wayland.rb [hunk 2]
```diff
@@ -3,38 +3,29 @@
...
+    system "./configure \
+            #{CREW_OPTIONS} \
+            --disable-documentation"
+    system 'make'
   end
 
   def self.install
-    system "make", "DESTDIR=#{CREW_DEST_DIR}", "install"
+    system 'make', "DESTDIR=#{CREW_DEST_DIR}", 'install'
+    system 'libtool --finish #{CREW_LIB_PREFIX}'
```
Comment (uberhacker) [inline packages/wayland.rb:29]: Please enclose in double quotes (") and test first.

**Comment by satmandu** [comment]
Is there a reason for updating just the package file with a new macro without generating new binaries when there is not a new version?

For instance, here's my not uploaded version of this with the newer beta release using meson/ninja:

```
require 'package'

class Wayland < Package
  description 'Wayland is intended as a simpler replacement for X, easier to develop and maintain.'
  homepage 'https://wayland.freedesktop.org'
  version '1.18.91'
  compatibility 'all'
  source_url 'https://wayland.freedesktop.org/releases/wayland-1.18.91.tar.xz'
  source_sha256 '102cb3511d4f923924ea5a969bbb801fff32662da933711ca614ebefb481b3d1'

  binary_url ({
     aarch64: 'file:///usr/local/tmp/packages/wayland-1.18.91-chromeos-armv7l.tar.xz',
      armv7l: 'file:///usr/local/tmp/packages/wayland-1.18.91-chromeos-armv7l.tar.xz',
        i686: 'file:///usr/local/tmp/packages/wayland-1.18.91-chromeos-i686.tar.xz',
      x86_64: 'file:///usr/local/tmp/packages/wayland-1.18.91-chromeos-x86_64.tar.xz',
  })
  binary_sha256 ({
     aarch64: 'c31016a8d99a195f8784d96ccd196e921387a3694889894c769c72ade1221d9d',
      armv7l: 'c31016a8d99a195f8784d96ccd196e921387a3694889894c769c72ade1221d9d',
        i686: '927c0dae13bd95a4f75124498e97cde02345d343fbe8012055110212c9d70edd',
      x86_64: '0dc0f3f3188a024afd6d4446109d9a2474b1d842e8323f5af9118428bdd3a927',
  })

 depends_on 'libpng'
 depends_on 'libxslt'
 depends_on 'libffi'
 depends_on 'graphviz' => :build # GraphViz doesn't have PNG support enabled

  def self.build
    system "meson #{CREW_MESON_LTO_OPTIONS} \
    -Ddocumentation=false \
    builddir"
    system "meson configure builddir"
    system "ninja -C builddir"
  end

  def self.install
    system "DESTDIR=#{CREW_DEST_DIR} ninja -C builddir install"
  end
end
```

File: packages/wayland.rb [hunk 3]
```diff
@@ -21,20 +21,25 @@ class Wayland < Package
      x86_64: 'bf28cacca6c316fba02ddb76bcd029034bedfd0ca3537231efce89b1ab81aa09',
   })
```
Comment (uberhacker) [inline packages/wayland.rb:22]: Can we remove the binaries?

File: packages/wayland.rb (hunk 2 above)
Comment (supechicken) [reply packages/wayland.rb]: Done

File: packages/wayland.rb (hunk 3 above)
Comment (supechicken) [reply packages/wayland.rb]: Done

**Pull request merged**